spec/integration/guest_view_spec.rb

```diff
@@ -3,11 +3,9 @@
 describe "the guest view", type: :feature do
   describe "the home page" do
 
-    before(:each) do
```

Comment:
I think the main reason to remove the block here is for readability. The test file spans almost a hundred lines, so having everything in each test adds readability.